<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>396</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TextoPregunta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>731</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Aquí viene texto de la pregunta</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonOpcion1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Opcion1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Campo1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Categoría:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextoCategoria">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonOpcion2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Opcion2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonOpcion3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Opcion3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonOpcion4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Opcion4</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextoInfo">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>240</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>¡Muy bien! Acabas de empezar el juego, intenta pasar esta pregunta. Recuerda que recibes 20 puntos por pregunta respondida correctamente.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="BarraPuntaje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="format">
     <string>%p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonRegresar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Regresar a la ventana anterior</string>
    </property>
   </widget>
   <widget class="QLabel" name="Campo2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Puntaje:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Campo3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Puntos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
